go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/vroomy/plugins
+
+go 1.13
```

Comment:
Do we want to require 1.14 for build compatibility?

Reply:
1.14 is still RC, can you test locally and see if it breaks?